app/views/items/_item.html.erb

```diff
@@ -1,3 +1,4 @@
 <div class="col-md-6">
     <%= item.name %>
+    <%= link_to "", item, method: :delete, class: 'glyphicon glyphicon-ok' %>
```

Comment:
Okay, when adding this in, it isn't working. I am getting an error:

```
Showing /Users/nicoledelarosa/code/blocitoff/app/views/items/_item.html.erb where line #3 raised:

undefined method `items_path' for #<#<Class:0x007fb50b4a4508>:0x007fb509be3ed8>
Extracted source (around line #3):
1
2
3
4
              
<div class="col-md-6">
    <%= item.name %>
    <%= link_to " ", item, method: :delete, class: 'glyphicon glyphicon-ok' %>
</div>
```

I think there may be something wrong with my routes..? Any ideas? (I think this is tied to my issue with my spec test as well.